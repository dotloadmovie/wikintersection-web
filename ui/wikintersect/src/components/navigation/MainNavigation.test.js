import React from 'react';
import MainNavigationContainer from './MainNavigation.container';

import { shallow } from 'enzyme';
it('displays the MainNavigation correctly', () => {
    const component = shallow(<MainNavigationContainer/>)

    expect(component).toMatchSnapshot();
})
