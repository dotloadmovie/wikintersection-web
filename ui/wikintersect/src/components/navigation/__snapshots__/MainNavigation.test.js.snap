// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the MainNavigation correctly 1`] = `
<Menu
  className=""
  defaultSelectedKeys={
    Array [
      "1",
    ]
  }
  focusable={false}
  mode="horizontal"
  style={
    Object {
      "lineHeight": "64px",
    }
  }
  theme="dark"
>
  <MenuItem
    key="1"
  >
    Search
  </MenuItem>
  <MenuItem
    key="2"
  >
    About
  </MenuItem>
</Menu>
`;
